canonical-web-and-design/ubuntu.com
# Add redirect for /engage/finance

## Done

- redirect for /engage/finance to /engage/ubuntu-finance-sector-whitepaper

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/finance
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are sent to http://0.0.0.0:8001/engage/ubuntu-finance-sector-whitepaper
- **Update the path on [discourse](https://discourse.ubuntu.com/t/engage-pages-and-takeovers/17229) from `/engage/finance` to `/engage/ubuntu-finance-sector-whitepaper`** as you release the site

## Issue / Card

Fixes #8788
